<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicWebMusicRadioWidget</class>
 <widget class="QWidget" name="MusicWebMusicRadioWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>网络电台播放窗口</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>475</width>
     <height>335</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>网络电台播放窗口</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>473</x>
     <y>22</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="playButton">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>245</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="MusicClickedSlider" name="volumeSlider">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>287</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>音量</string>
   </property>
  </widget>
  <widget class="QWidget" name="backgroundMask" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>45</y>
     <width>475</width>
     <height>310</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(255,255,255,150)</string>
   </property>
  </widget>
  <widget class="QLabel" name="lrcLabel">
   <property name="geometry">
    <rect>
     <x>209</x>
     <y>50</y>
     <width>270</width>
     <height>290</height>
    </rect>
   </property>
  </widget>
  <widget class="MusicRoundAnimationLabel" name="artistLabel" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>85</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="previousButton">
   <property name="geometry">
    <rect>
     <x>108</x>
     <y>245</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>245</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="positionLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20px;font-weight:bold;color:white</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="durationLabel">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>310</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15px;font-weight:bold;color:white;</string>
   </property>
  </widget>
  <widget class="MusicMarqueeWidget" name="titleWidget" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>backgroundMask</zorder>
  <zorder>topTitleName</zorder>
  <zorder>topTitleCloseButton</zorder>
  <zorder>playButton</zorder>
  <zorder>volumeSlider</zorder>
  <zorder>label_2</zorder>
  <zorder>lrcLabel</zorder>
  <zorder>artistLabel</zorder>
  <zorder>previousButton</zorder>
  <zorder>nextButton</zorder>
  <zorder>positionLabel</zorder>
  <zorder>durationLabel</zorder>
  <zorder>titleWidget</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MusicClickedSlider</class>
   <extends>QSlider</extends>
   <header>musicclickedslider.h</header>
  </customwidget>
  <customwidget>
   <class>MusicMarqueeWidget</class>
   <extends>QWidget</extends>
   <header>musicmarqueewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MusicRoundAnimationLabel</class>
   <extends>QWidget</extends>
   <header>musicroundanimationlabel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
